tests/adapters/switches/brocade_backward_compatibility_test.py

```diff
@@ -107,23 +108,23 @@ def test_remove_trunk_vlan_accepts_no_ethernet(self):
 
         self.switch.remove_trunk_vlan("1/11", vlan=2999)
 
-    def test_shutdown_interface_accepts_no_ethernet(self):
+    def test_set_interface_state_OFF_accepts_no_ethernet(self):
         self.shell_mock.should_receive("do").with_args("configure terminal").once().ordered().and_return([])
         self.shell_mock.should_receive("do").with_args("interface ethernet 1/4").and_return([]).once().ordered()
         self.shell_mock.should_receive("do").with_args("disable").and_return([]).once().ordered()
         self.shell_mock.should_receive("do").with_args("exit").and_return([]).twice().ordered().ordered()
 
-        self.switch.shutdown_interface("1/4")
+        self.switch.set_interface_state("1/4", OFF)
 
-    def test_openup_interface_accepts_no_ethernet(self):
+    def test_set_interface_state_ON_accepts_no_ethernet(self):
```

Comment:
I thought of it... but for me it would look like the preposition... I don't mind making the switch though